src/store/Slices/sidebarSlice.ts

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const initialState = {
+  show: true,
+}
+
+const sidebarSlice = createSlice({
```

Comment:
There is no need to use redux for the sidebar state. I also suggest storing the sidebar state in the localStorage, so that when user refreshes the page gets the preferred state before refreshing.